src/sass/layouts/_sidebar.scss

```diff
@@ -6,6 +6,11 @@ header{
     flex-basis: 20%;
 }
 
+
+%pointer-event{
+    pointer-events: none;
+}
```

Comment:
`pointer-event` has many properties, you can name the placeholder to suit the `none` better

Reply:
sure. i will change it